README.md

```diff
@@ -1,36 +1,38 @@
-# Golang 101 workshop
-You will need to install golang before you can play with this.
+# Go 101 workshop
 
+Practical intro to Go programming language.
 
-## golang installation linux
-##  download
-```
-wget https://dl.google.com/go/go1.11.linux-amd64.tar.gz
-sudo tar -C /usr/local -xzf go1.11.linux-amd64.tar.gz
-mkdir ~/go
-```
+---
 
-## set paths
-add these paths to your bash/zsh profile
+Everyone wants a web site, but did you know you can have a fully-fledged HTTP server in less than 20 lines of Go?
 
-```
-export GOPATH=/home/$USER/go
-export GOROOT=/usr/local/go
-export GOBIN=/home/$USER/go/bin/
-```
-after you reload your terminal profile, you should be able to fetch go packages using go.
-```
-go get golang.org/x/tools/cmd/goimports
-```
+```go
+package main
 
-# go package management
+import (
+    "fmt"
+    "log"
+    "net/http"
+)
 
+func handler(w http.ResponseWriter, r *http.Request) {
+    fmt.Fprintf(w, "Hi there, I love %s!", r.URL.Path[1:])
+}
 
+func main() {
+    http.HandleFunc("/", handler)
+    log.Fatal(http.ListenAndServe(":8080", nil))
+}
 ```
-go get -v github.com/egidijus/golang-101-workshop
-```
 
+## Requirements
+
+Follow the steps below to install Go on your machine:
+
+- Download binaries - [https://golang.org/dl/](https://golang.org/dl/)
```

Comment:
I want to test people installing go, it's easy to take installation for granted once you know how it works and why it works.

Reply:
Why test that? I would prefer spending time on hacking together some code rather than people just messing arround with their terminals 😮